docs/migration/README.rst

```diff
@@ -236,7 +246,65 @@ Once these changes are merged into app-interface, you should be able to open a
 PR against the app's source code repo and see Bonfire deploy the app, assuming
 all dependent services are also set up with Bonfire.
 
-.. _Bonfire: https://github.com/redhatinsights/bonfire 
+
+
+Testing your migration code/template changes
+-----------------------------------
+
+**Testing Without Jenkins**
+
+
+You can test the changes you are making to an application before any Jenkins jobs are wired up to
+your app's repo. This allows you to test changes before they are merged into the app's repository
+or app-interface. The only thing you *cannot* test is of course whether the Jenkins jobs work as
+expected but you could even run ``pr_check.sh`` locally if you pass all the proper environment
+variables to it.
+
+#. Create a separate branch on the app repo and add:
+    * the new deployment template defining your ``ClowdApp``
+    * whatever code changes are needed so that your app can run as a ClowdApp
+    * ``build_deploy.sh``
+    * ``pr_check.sh``
+
+#. Push your changes to git and note the git commit's hash.
+
+#. Create a separate branch in app-interface and add:
+    * an updated deploy.yaml/saas file that configures the ephemeral deployment target (make sure
+      that the template path for the app points to your new ClowdApp template)
+
+#. Clone `Bonfire`_ and install it
+
+#. | Follow `these directions <https://github.com/RedHatInsights/bonfire#running-a-local-qontract-server>`_
+    to set up your own local app-interface server and start it
+
+#. Build your app's new docker image and somehow push it to quay with a unique image tag.
```

Comment:
Right now the only suggestion I have is to create your own repo, `docker build`/`docker push` the image there, and temporarily edit your app template to point the app at your repo instead of `quay.io/cloudservices`

Added a note on this in the doc